app/cypress.config.ts

```diff
@@ -0,0 +1,20 @@
+import {defineConfig} from 'cypress'
+
```

Comment:
cypress 12 detected the old version files and offered me to migrate them automatically which involved renaming the test files and config for cypress - this is the result and it seems to work without issues